src/utils/index.js

```diff
@@ -24,3 +24,9 @@ export {
   loginFailedActions,
   signoutUser,
 } from "utils/login-utils";
+export {
+  compose,
+  sortByDate,
+  filterByPriority,
+  filterByTags,
+} from "./filter-utils";
```

Comment:
Instead of collecting all the items from "./filter-utils" and then exporting it in index.js you can simply write it as;
```
export * from "./filter-utils"
```
It would work in the same manner as this one

Reply:
This was a great suggestion, thanks!